src/panels/config/logs/error-log-card.ts

```diff
@@ -378,6 +425,18 @@ class ErrorLogCard extends LitElement {
         isComponentLoaded(this.hass, "hassio") &&
         this.provider
       ) {
+        // check if there are any logs at all
+        const testResponse = await fetchHassioLogs(
+          this.hass,
+          this.provider,
+          `entries=:-1:`,
+          this._boot
+        );
+        const testLogs = await testResponse.text();
+        if (!testLogs.trim()) {
+          this._loadingState = "empty";
+        }
```

Comment:
Should we return if there is no logs?

Reply:
No! Because the stream will run and there can occure logs afterwards.